docs/source/existing_code.rst

```diff
@@ -0,0 +1,174 @@
+
+   import frobnicate  # type: ignore
+   ...
+   frobnicate.initialize()  # OK (but not checked)
+
+You can also use a mypy configuration file, which is convenient if
+there are a large number of errors to ignore. For example, to disable
+errors about importing ``frobnicate`` and ``acme`` everwhere in your
+codebase, use a config like this:
+
+.. code-block:: text
+
+   [mypy-frobnicate.*]
+   ignore_missing_imports = True
+
+   [mypy-acme.*]
+   ignore_missing_imports = True
+
+You can add multiple sections for different modules that should be
+ignored.
+
+If your config file is named ``mypy.ini``, this is how you run mypy:
+
+.. code-block:: text
+
+   mypy --config-file mypy.ini mycode/
+
+If you get a large number of errors, you may want to ignore all errors
+about missing imports.  This can easily cause problems later on and
+hide real errors, and it's only recommended as a last resort. We
+don't explain this option in more detail here.
+
+Mypy follows imports by default. This can result in a few files passed
+on the command line causing mypy to process a large number of imported
+files, resulting in lots of errors you don't want to deal with at the
+moment. There is a config file option to disable this behavior, but
+since this can hide errors, it's not recommended for most users.
+
+Mypy runner script
+------------------
+
+Introduce a mypy runner script that runs mypy, so that every developer
+will use mypy consistently. Here are some things you may want to do in
+the script:
+
+* Ensure that the correct version of mypy is installed.
+
+* Specify mypy config file or command-line options.
+
+* Provide set of files to type check. You may want to implement
+  inclusion and exclusion filters for full control of the file
+  list.
+
+Continuous Integration
+----------------------
+
+Once you have a clean mypy run and a runner script for a part
+of your codebase, set up your Continuous Integration (CI) system to
+run mypy to ensure that developers won't introduce bad annotations.
+A simple CI script could look something like this:
+
+.. code-block:: text
+
+    python3 -m pip install mypy==0.600  # Pinned version avoids surprises
+    scripts/mypy  # Runs with the correct options
+
+Annotate library modules
+------------------------
+
+Most projects have some commonly used library code. It's a good idea
+to annotate these pretty early on, since this allows code using these
+modules to be type checked more effectively. Since mypy supports
+gradual typing, it's okay to leave some unannotated library
+modules. The more you annotate, the more useful mypy will be, but even
+a little annotation coverage is useful.
+
+Write annotations as you go
+---------------------------
+
+Now you are ready to include type annotations in your development
+workflows. Establish in your code style conventions that developers
+should add annotations for any new code, and it's also a good idea to
+write annotations when you modify existing code.  This way you'll
```

Comment:
I think this sentence would flow a bit more cleanly if we pulled it out into bullet points. E.g. something like:

> Establish in your code style conventions that developers:
>
> 1. Should add annotations when adding new code
> 2. Are encouraged to add annotations when modifying existing code
>
> That way you'll...

This is sort of subjective though, so feel free to ignore this suggestion.

Also, maybe "should" -> "must"? Or maybe that's too harsh?